libats/src/main/scala/com/advancedtelematic/libats/data/DataType.scala

```diff
@@ -10,7 +10,10 @@ import scala.language.postfixOps
 
 object DataType {
 
-  final case class Namespace(get: String) extends AnyVal
+  final case class Namespace(get: String) extends AnyVal {
```

Comment:
Are you sure it is possible? We have namespaces already which are not UUIDs

Reply:
What I understood from Jerry is that we want to start making the namespaces UUIDs while still supporting the old namespaces. I figured starting to use this `generate` method when we need a new namespace ought to be enough, or am I missing something?

Reply:
Sorry, my bad. Need more sleep. :)
I would move the generate method into a companion object though. And probably making it an urn like correlation id

Reply:
Makes sense. I've updated it.